src/components/refresher/refresher.ts

```diff
@@ -202,6 +202,7 @@ export class Refresher {
 
   constructor(private _plt: Platform, @Host() private _content: Content, private _zone: NgZone, gestureCtrl: GestureController) {
     this._events = new UIEventManager(_plt);
+    _content._hasRefresher = true;
     _content.setElementClass('has-refresher', true);
```

Comment:
@manucorporat Yes would be nice, maybe we could even use this on `Content` and remove the `setElementClass` completely, so there's no need for a setter function?

```ts
host {
  '[class.has-refresher]': '_hasRefresher'
}
```